benchopt/plotting/html/templates/benchmark.mako.html

```diff
@@ -111,37 +109,37 @@ <h1>
         % endif
       </div>
       % endfor
+
     </td>
     <td class="sysinfo">
 
-      % for idx_info, all_info in enumerate(result['sysinfo']['main'].values()):
-      <div class="all_info">
-        <div class="info">
-          % for idx in range(len(all_info)):
-          <b>${list(all_info)[idx]}</b>: ${list(all_info.values())[idx]}
+      <ul style="list-style-type: none; margin-top: 0;">
+      % for idx_info, (keys_info, vals_info) in enumerate(result['sysinfo']['main'].items()):
+        % if vals_info != "":
+          <li>
+            <b>${keys_info}</b>: ${vals_info}
```

Comment:
You can remove the `enumerate` and this is only `key, val` no? not lists?
```suggestion
      % for name, val in result['sysinfo']['main'].items():
        % if val != "":
          <li>
            <b>${name}</b>: ${val}
```

Reply:
I'm currently looking for a way to remove this enumerate. Because I don't like that the + icon top-right of the cell to expand is index-dependant.

Reply:
Now it's better :slightly_smiling_face: